packages/graphql/src/resolvers/IdentityEvents.js

```diff
@@ -26,7 +26,8 @@ const attestationProgressPct = {
   kakao: 10,
   github: 10,
   linkedin: 10,
-  wechat: 10
+  wechat: 10,
```

Comment:
The sum of all those is now 120% :)
Make each of those 100/12

Reply:
@franckc This shouldn't be a problem, since we cap the strength at 100% https://github.com/OriginProtocol/origin/blob/a8861d8c85f632dbc6f7d2393c5b2f4839b5951d/packages/graphql/src/resolvers/IdentityEvents.js#L300-L306